roles/plugins/foreman_openscap/tasks/main.yml

```diff
@@ -28,21 +28,54 @@
     {{ foreman_openscap_hammer }} proxy import-classes --name {{ foreman_openscap_proxy_name }}
   tags:
     - seed
-  when: '{{ foreman_openscap_proxy_name != False }}'
 
 - name: 'Import default scap content'
   shell: >
     foreman-rake foreman_openscap:bulk_upload:default
   tags:
     - seed
-  when: '{{ foreman_openscap_proxy_name != False }}'
-
-# TODO: disabled until cli is fixed
-# - name: 'Create example policy'
-#   shell: >
-#     {{ foreman_openscap_hammer }} policy create --name 'Example policy'
-#      --period weekly --weekday monday
-#      --scap-content 'Red Hat centos6 default content'
-#      --scap-content-profile-id TODO
-#   tags:
-#     - seed
+
+# Get the smart proxy ID of the local katello:
```

Comment:
Is the comment here needed? I'd think the `name` parameter should capture all of the relevant info and thus be in code.

Reply:
nope, probably copied from elsewhere